### DISCUSSION TITLE
How to start with existing ckpt file?

### DISCUSSION THREAD

**Discussion by jonmillerschwartz** [TheLastBen/fast-stable-diffusion#339] [General]
I'm having trouble figuring out how to load an existing ckpt that's already in my Drive. Any tips would be appriciated!

**Comment by TheLastBen** [comment]
Use the colab file explorer in the left to copy the ckpt path and paste it in the "path_to_model" in the model download cell

**Comment by EricBintner** [comment]
Related question: Does it matter that the training file is half-precision? Would it be better if we trained from 1-5, or does it really matter?

**Comment by TheLastBen** [comment]
I didn't notice any difference between half and full precision training, but that's just me, you can change the option in the cell code if you want. What do you mean with 1-5 ?
